### ISSUE TITLE
Unable to get() object with custom properties

### ISSUE THREAD

**Issue by FearonChris** [oasis-open/cti-python-stix2#87]
```
File "/usr/local/lib/python2.7/dist-packages/stix2/environment.py", line 110, in get
    return self.source.get(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/stix2/sources/__init__.py", line 262, in get
    data = ds.get(stix_id=stix_id, _composite_filters=all_filters)
  File "/usr/local/lib/python2.7/dist-packages/stix2/sources/__init__.py", line 262, in get
    data = ds.get(stix_id=stix_id, _composite_filters=all_filters)
  File "/usr/local/lib/python2.7/dist-packages/stix2/sources/filesystem.py", line 156, in get
    all_data = self.query(query=query, _composite_filters=_composite_filters)
  File "/usr/local/lib/python2.7/dist-packages/stix2/sources/filesystem.py", line 294, in query
    stix_objs = [parse(stix_obj_dict) for stix_obj_dict in all_data]
  File "/usr/local/lib/python2.7/dist-packages/stix2/core.py", line 94, in parse
    return obj_class(allow_custom=allow_custom, **obj)
  File "/usr/local/lib/python2.7/dist-packages/stix2/base.py", line 104, in __init__
    raise ExtraPropertiesError(cls, extra_kwargs)
stix2.exceptions.ExtraPropertiesError: Unexpected properties for Vulnerability:
```

**Comment by clenk** [comment]
Thank you for this bug report, @FearonChris. The upcoming changes to fix #76 should fix this issue as well.

**Comment by gtback** [comment]
Thanks, @FearonChris. We're working on this in conjunction with our work on #76.

**Comment by FearonChris** [comment]
Apologies btw, I feel like i'm bombarding you guys! Thanks for your prompt responses and efforts on this project.

**Comment by gtback** [comment]
No apologies needed, @FearonChris. Please keep the issues coming! It's good to see people using the libraries and exploring use cases (and code paths) that we haven't thought of (especially regarding custom properties, where it's harder for us to know the breadth of use cases).

Issue #76 led us to rethink how we are doing custom properties in the DataStore classes, which I think will result in a better overall design.

**Comment by gtback** [comment]
Fixed by #88
